pkgs/development/python-modules/django-picklefield/default.nix

```
@@ -1,16 +1,25 @@
-{ lib, buildPythonPackage, fetchPypi, django }:
+{ lib, buildPythonPackage, fetchFromGitHub, django }:
 
 buildPythonPackage rec {
   pname = "django-picklefield";
   version = "3.0.1";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "15ccba592ca953b9edf9532e64640329cd47b136b7f8f10f2939caa5f9ce4287";
+  src = fetchFromGitHub {
+    owner = "gintas";
+    repo = "django-picklefield";
+    rev = "4d5e3b2995df6597d45ad59d92ead38abc5c34af";
```

please use official releases
```suggestion
    rev = "v${version}";
```
https://github.com/gintas/django-picklefield/releases/tag/v3.0.1

---

Oh, I didn't realize I could batch suggestions. I've incorporated both in now.